presto-main/src/main/java/com/facebook/presto/util/GraphvizPrinter.java

```diff
@@ -241,7 +241,7 @@ public Void visitSample(SampleNode node, Void context)
         @Override
         public Void visitSort(SortNode node, Void context)
         {
-            printNode(node, format("Sort[%s]", Joiner.on(", ").join(node.getOrderBy())), NODE_COLORS.get(NodeType.SORT));
+            printNode(node, format("Sort[%s]", Joiner.on(", ").join(node.getOrderingScheme().getOrderBy())), NODE_COLORS.get(NodeType.SORT));
```

Comment:
Consider extracting formatting of `node.getOrderingScheme()` to a separate method. I understand introduction of `OrderingScheme` is all about reuse, so separate method helps reuse.

Reply:
TBH I am not sure how `GraphvizPrinter` is used, so I prefer to leave formatting as it was.